torch/csrc/jit/import_source.cpp

```diff
@@ -133,22 +121,39 @@ struct SourceImporter {
           << "expected an integral version but found " << version.text();
     return size_t(version.asIntegral());
   }
-};
+  // Our tests generate a bunch of code where the version number is not
+  // set. In this case, we just choose the most up-to-date version.
+  // Actual archives should always set the version.
+  return CURRENT_OP_VERSION_SET;
```

Comment:
This looks like a bad idea to me. We should fix the test to not do dumb things, not make our code do them.